test/e2e-encryption-kms/encryption_kms.go

```
@@ -41,12 +40,12 @@ var _ = g.Describe("[sig-openshift-apiserver] cluster-openshift-apiserver-operat
 // 9. Disables encryption (Identity) again
 // 10. Verifies token is NOT encrypted again
 func testKMSEncryptionOnOff(ctx context.Context, t testing.TB) {
-	cs := operatorencryption.GetClients(t)
+	cs := library.GetClients(t)
 
 	ns := fmt.Sprintf("test-kms-encryption-on-off-%d", rand.IntN(4))
-	_, err := cs.KubeClient.CoreV1().Namespaces().Create(ctx, &corev1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: ns}}, metav1.CreateOptions{})
+	_, err := cs.Kube.CoreV1().Namespaces().Create(ctx, &corev1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: ns}}, metav1.CreateOptions{})
 	require.NoError(t, err)
-	defer cs.KubeClient.CoreV1().Namespaces().Delete(ctx, ns, metav1.DeleteOptions{})
+	defer cs.Kube.CoreV1().Namespaces().Delete(ctx, ns, metav1.DeleteOptions{})
```

golangci-lint (2.12.2)

[error] Error return value of `(k8s.io/client-go/kubernetes/typed/core/v1.NamespaceInterface).Delete` is not checked

(errcheck)